docs/developers/docs/internationalization.md

```diff
@@ -8,15 +8,16 @@ Pupilfirst LMS uses _Rails Internationalization_ (I18n) API for internationaliza
 
 ## Organization of en.yml
 
-| Key                      |                       Value                       |
-| ------------------------ | :-----------------------------------------------: |
-| `components.COMPONENT.*` |            Component-specific strings.            |
-| `jobs.[MODULE].JOB.*`    |               Strings used by jobs.               |
-| `models.MODEL.COLUMN.*`  | Translation of database values to display values. |
-| `mutations.MUTATION.*`   |            Strings used by mutations.             |
-| `queries.QUERY.*`        |      Strings used by resolvers and mutators.      |
-| `shared.*`               |                  Shared strings.                  |
-| `CONTROLLER.ACTION.*`    |        Strings used in traditional views.         |
+| Key                               |                       Value                       |
+| --------------------------------- | :-----------------------------------------------: |
+| `components.COMPONENT.*`          |            Component-specific strings.            |
+| `controllers.CONTROLLER.ACTION.*` |           Controller-specific strings.            |
+| `jobs.[MODULE].JOB.*`             |               Strings used by jobs.               |
+| `models.MODEL.COLUMN.*`           | Translation of database values to display values. |
+| `mutations.MUTATION.*`            |            Strings used by mutations.             |
+| `queries.QUERY.*`                 |      Strings used by resolvers and mutators.      |
+| `shared.*`                        |                  Shared strings.                  |
+| `CONTROLLER.ACTION.*`             |        Strings used in traditional views.         |
```

Comment:
@prashanth726 This last line is no longer required since the new L14 covers this.